odoo/addons/base/views/ir_qweb_widget_templates.xml

```diff
@@ -37,7 +37,7 @@
             <div t-if="phone and 'phone' in fields"><i t-if="not options.get('no_marker') or options.get('phone_icons')" class='fa fa-phone fa-fw' role="img" aria-label="Phone" title="Phone"/> <span class="o_force_ltr" itemprop="telephone" t-esc="phone"/></div>
             <div t-if="mobile and 'mobile' in fields"><i t-if="not options.get('no_marker') or options.get('phone_icons')" class='fa fa-mobile fa-fw' role="img" aria-label="Mobile" title="Mobile"/> <span class="o_force_ltr" itemprop="telephone" t-esc="mobile"/></div>
             <!-- Prevent issue with iOS Safari parsing of schema data without telephone itemprops -->
-            <div t-elif="not (phone and 'phone' in fields)" itemprop="telephone"/>
+            <div t-if="not (phone and 'phone' in fields)" itemprop="telephone"/>
```

Comment:
The reason I used `t-elif` is to not have to check the `mobile` values again.

I don't think this approach fixes it, but if it does, the conditional should be
```suggestion
            <div t-if="not (phone and 'phone' in fields or mobile and 'mobile' in fields)" itemprop="telephone"/>
```

Reply:
Yeah right, for me testing in local is fine, it does fix the problem that mention in the issue above